samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/transition.c

```diff
@@ -0,0 +1,456 @@
+
+	resolution = (state->tt >> 6);
+	steps_multiplier = state->tt & 0x3F;
+
+	state->tt_counter = tt_counter_calculator(steps_multiplier, resolution);
+
+	state->tt_delta = ((float) state->last_delta / state->tt_counter);
+}
+
+void move_level_transition_time_values(struct generic_level_state *state)
+{
+	u8_t steps_multiplier, resolution;
+
+	resolution = (state->tt >> 6);
+	steps_multiplier = state->tt & 0x3F;
+
+	state->tt_counter = tt_counter_calculator(steps_multiplier, resolution);
+
+	state->tt_delta = ((float) state->last_delta / state->tt_counter);
+}
+
+void light_lightnes_actual_tt_values(struct light_lightness_state *state)
+{
+	u8_t steps_multiplier, resolution;
+
+	resolution = (state->tt >> 6);
+	steps_multiplier = state->tt & 0x3F;
+
+	state->tt_counter = tt_counter_calculator(steps_multiplier, resolution);
+
+	state->tt_delta = ((float) abs(state->actual - state->target_actual)
+			   / state->tt_counter);
+
+	if (state->target_actual < state->actual) {
+		state->tt_delta *= -1;
+	}
+}
+
+void light_lightnes_linear_tt_values(struct light_lightness_state *state)
+{
+	u8_t steps_multiplier, resolution;
+
+	resolution = (state->tt >> 6);
+	steps_multiplier = state->tt & 0x3F;
+
+	state->tt_counter = tt_counter_calculator(steps_multiplier, resolution);
+
+	state->tt_delta = ((float) abs(state->linear - state->target_linear)
+			   / state->tt_counter);
+
+	if (state->target_linear < state->linear) {
+		state->tt_delta *= -1;
+	}
+}
+
+void light_ctl_tt_values(struct light_ctl_state *state)
+{
+	u8_t steps_multiplier, resolution;
+
+	resolution = (state->tt >> 6);
+	steps_multiplier = state->tt & 0x3F;
+
+	state->tt_counter = tt_counter_calculator(steps_multiplier, resolution);
+
+	state->tt_lightness_delta =
+		((float) abs(state->lightness - state->target_lightness)
+		 / state->tt_counter);
+
+	state->tt_temp_delta =
+		((float) abs(state->temp - state->target_temp)
+		 / state->tt_counter);
+
+	state->tt_delta_uv_delta =
+		((float) abs(state->delta_uv - state->target_delta_uv)
+		 / state->tt_counter);
+
+	if (state->target_lightness < state->lightness) {
+		state->tt_lightness_delta *= -1;
+	}
+
+	if (state->target_temp < state->temp) {
+		state->tt_temp_delta *= -1;
+	}
+
+	if (state->target_delta_uv < state->delta_uv) {
+		state->tt_temp_delta *= -1;
+	}
+}
+
+void light_ctl_temp_tt_values(struct light_ctl_state *state)
+{
+	u8_t steps_multiplier, resolution;
+
+	resolution = (state->tt >> 6);
+	steps_multiplier = state->tt & 0x3F;
+
+	state->tt_counter = tt_counter_calculator(steps_multiplier, resolution);
+
+	state->tt_temp_delta = ((float)
+			   abs(state->temp - state->target_temp)
+				/ state->tt_counter);
+
+	state->tt_delta_uv_delta =
+		((float) abs(state->delta_uv - state->target_delta_uv)
+		 / state->tt_counter);
+
+	if (state->target_temp < state->temp) {
+		state->tt_temp_delta *= -1;
+	}
+}
+
+void lightness_transition_timer_handler(struct k_timer *dummy)
+{
+	s32_t tmp_s32;
+	u32_t tmp_u32;
```

Comment:
Move these to the smaller scopes where they're use and rename them more descriptively.

Reply:
Same as per above